gen/build_deploy/powershell/dcos_install.ps1

```diff
@@ -0,0 +1,422 @@
+<#
```

Comment:
can we delete [gen/build_deploy/powershell/dcos_install.ps1.in](gen/build_deploy/powershell/dcos_install.ps1.in) then?

Reply:
There are only 4 lines different. I prefer to not copy whole file just to change mandatory parameter setting.

```diff
# diff gen/build_deploy/powershell/dcos_install.ps1{,.in}                                          
45a46,48
> # Metadata:
> #   dcos image commit : {{ dcos_image_commit }}
> #   generation date   : {{ generation_date }}
51,52c54,55
<     [Parameter(Mandatory=$true)] [string] $bootstrap_url,
<     [Parameter(Mandatory=$true)] [string] $masters,
---
>     [Parameter(Mandatory=$false)] [string] $bootstrap_url = '{{ bootstrap_url }}',
>     [Parameter(Mandatory=$false)] [string] $masters = '{{ master_list }}',
```

Reply:
Within GA scope dcos_install.ps1 was replaced in favor of template. Extra steps were done to make this filled up in Python. Now, to full fill your request it needs to be done in Powershell - use a new template, but make a substitution within a build_genconf_windows.ps1 . 
However this work will be redundant in GA